lib/logstash/inputs/udp.rb

```diff
@@ -97,10 +101,12 @@ def inputworker(number)
           decorate(event)
           event.set("host", client[3]) if event.get("host").nil?
           @output_queue.push(event)
+          metric.increment(:events)
```

Comment:
As I shared at https://github.com/logstash-plugins/logstash-input-http/pull/58#discussion_r72979337, happy to understand what is under your mind. Can you elaborate a bit more your thoughts?